llvm/bindings/ocaml/debuginfo/debuginfo_ocaml.c

```diff
@@ -972,7 +972,7 @@ value llvm_dibuild_create_parameter_variable_bytecode(value *argv, int arg) {
 value llvm_dibuild_insert_declare_before_native(value Builder, value Storage,
                                                 value VarInfo, value Expr,
                                                 value DebugLoc, value Instr) {
-  LLVMValueRef Value = LLVMDIBuilderInsertDeclareBefore(
+  LLVMValueRef Value = LLVMDIBuilderInsertDeclareRecordBefore(
```

Comment:
Shouldn't this stay as the base version, so that it transitions to inserting Records at the same time as the C-API?

Reply:
Yep, mistakenly update here. Done.